instrumentserver/dashboard/__init__.py

```diff
@@ -0,0 +1,257 @@
+There can be multiple of them, each representing an individual plot,
+and they will be displayed in a row. Each plot will have its key as a name for it.
+Inside each plot there should be a dictionary for every parameter that should be displayed inside that specific plot.
+Each parameter will have its name as a key and must have a source_type and parameter_path.
+A few more options for parameters are available.
+
+Source_type:
+    only 2 options:
+        'parameter':
+            This indicates that the logger needs to ask for values to the instrumentserver.
+            If parameter is selected, interval should be specified in a separate options dictionary inside
+            the parameter dictionary (see example below) but is not necessary.
+            the default value is 1. the interval is in seconds.
+        'broadcast':
+            NOT IMPLEMENTED. Would pick up values from the automatic broadcast of parameter
+            changes from the instrumentserver. If broadcast is selected, in options, actions
+            should specify which updates to listen for. Default behaviour: all of them.
+
+parameter_path:
+    The string of the specified parameter with all of the submodules in.
+    The first submodule should always be the name of the instrument. Ex:'parameters.qubit.pipulse.len' ,
+    'triton.T1'
+
+server:
+    The location of the instrumentserver.
+
+port:
+    The port of the instrument server
+
+The options dictionary at the same level of the plot are where the settings for both the logger and the dashboard
+are located:
+
+    options:
+        This should be a dictionary at the same level of the plots in the dashboard.
+            refresh_rate
+                [int] This is both the amount of time, in seconds, between each time the logger saves data into the csv
+                and the dashboard reads the csv to show updates.
+                While the logger does record data for each individual parameter at its specified interval, it saves data
+                into storage all at once at this refresh_rate interval.
+            allowed_ip
+                [List[str]] List of the allowed ips that the server will show the dashboard
+                if ['*'], all ips are allowed and the server will show the dashboard to any browser that request it.
+            save_directory
+                [str] Directory of where the logger will save data. This should end with <name_of_file>.csv
+            load_directory
+                [str] Directory of where the dashboard will load data. This should end with <name_of_file>.csv
+            load_and_save
+                [str] Directory of where the dashboard will load and the logger save data.
+                If this option is present, the dashboard will override save_directory and load_directory options.
+                This should end with <name_of_file>.csv
+
+Example:
+^^^^^^^^
+
+The following is an example dictionary::
```

Comment:
i think we should adapt the config structure a bit:
i think it makes more sense to have entries 'plots' and 'options' (i.e., add one more level of nesting for the plots). i think it's too confusing to have one particular name ('options') not be a plot.